Description: fixing several simple warnings 
 fix includes path too
Author: Nicolas HOUDELOT (nicolas@demosdebs.org)
Last-Update: 2020-05-01
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/introsystem.h
+++ b/src/introsystem.h
@@ -23,8 +23,9 @@
 #include <stdlib.h>
 #include <math.h>
 
-#include "SDL.h"
-#include "SDL_opengl.h"
+#include <SDL/SDL.h>
+#include <SDL/SDL_opengl.h>
+#include <GL/glu.h>
 
 #include "bitmap.h"
 #include "data.h"
--- a/src/mikmod/mikmod.h
+++ b/src/mikmod/mikmod.h
@@ -523,7 +523,7 @@
 
 /* SDL_RWops compatability */
 #ifdef USE_RWOPS
-#include "SDL_rwops.h"
+#include <SDL/SDL_rwops.h>
 MIKMODAPI extern MODULE* Player_LoadRW(SDL_RWops*,int,BOOL);
 #endif /* USE_RWOPS */
 /* End SDL_RWops compatability */
--- a/src/sc2.c
+++ b/src/sc2.c
@@ -1,23 +1,24 @@
 /* Schtroumpf - the intro - (c) 2004 Gautier Portet < kassoulet @ no-log . org >
- *
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation; either version 2 of the License, or
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program; if not, write to the Free Software
- * Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
  */
 
 #include <stdio.h>
 #include <stdlib.h>
 #include <math.h>
+#include <sys/stat.h>
 
 #include "introsystem.h"
 #include "data.h"
@@ -97,7 +98,7 @@
 	/* Init */	
         puts("Schtroumph 2.0 (c) 2004 knights");
 #ifdef USE_SDL	
-	mkdir("pandora");
+	mkdir("pandora",ACCESSPERMS);
 #endif	
         OpenSystem();
         //OpenVideo(512,384,0);
